### PR TITLE
[WIP] renaming ParticipatePage2 to ParticipatePage

### PR DESCRIPTION
**Pull request by Pomax**
This is supposed to close https://github.com/mozilla/foundation.mozilla.org/issues/4775 but instead it's doing all the weirdness. See issue for the description of said weirdness.

### CODE REVIEW

File: network-api/networkapi/wagtailpages/pagemodels/base.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ class InitiativesPage(PrimaryPage):
     ]
 
 
-class ParticipatePage2(PrimaryPage):
+class ParticipatePage(PrimaryPage):
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/base.py:129]: This is the only "true" change, everything else is just changes to make sure references to the old name are now references to the new name.

File: network-api/networkapi/wagtailpages/pagemodels/base.py [hunk 2]
```diff
@@ -126,7 +126,7 @@ class InitiativesPage(PrimaryPage):
...
     parent_page_types = ['Homepage']
     template = 'wagtailpages/static/participate_page2.html'
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/base.py:131]: this has been left pointing to the original template until things actually work.

**Comment by KalobTaulien** [comment]
I tried this, this morning:

```
    class Meta:
        verbose_name = "Participate Page"
        verbose_name_plural = "Participate Pages"
        db_table = "wagtailpages_participatepage2"
        managed = False  # With and without the managed setting toggled off 
```

Ran into the same issue. I'm thinking we'll need to run a command to delete the original content type, cascade it through the tables (clean the db of this content type), then rename the `ParticipatePage2` to `ParticipatePage`

**Comment by Pomax** [comment]
let's retry this again at some later point.

**Pull request closed without merging**